### ISSUE TITLE
Is it possible to make it compatible with Portraits faces

### ISSUE THREAD

**Issue by DanielZanchi** [banjun/WatchFaceDumper#6]
When importing portrait faces I receive an error:

`error reading <NSFileWrapper: 0x600000066a80>: dataCorrupted(Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "face type", intValue: nil)], debugDescription: "Cannot initialize FaceType from invalid String value bundle", underlyingError: nil))`

Is it possible to make it compatible? 
Would like to create custom portrait faces with images that aren't shot on iPhone.

**Comment by banjun** [comment]
Yes I created the feature PR #3. You can try to import portrait watchfaces on the branch.

To edit, for now, 3 images per single photo should be created by the user and be placed by drag-drop.
These base, back, and mask images should have the expected color/format as described in the PR text.

**Comment by banjun** [comment]
like this https://twitter.com/banjun/status/1504449967922253824

**Comment by DanielZanchi** [comment]
Yes working good! thank you

**Comment by DanielZanchi** [comment]
I don't know why it's reading my images with a wrong resolution:
<img width="579" alt="image" src="https://user-images.githubusercontent.com/6886116/201907498-93f21320-ff86-4301-80aa-9a330af5e913.png">
[Archive.zip](https://github.com/banjun/WatchFaceDumper/files/10012155/Archive.zip)

**Comment by toyinswift** [comment]
@DanielZanchi @banjun Please can you show a code example of how to export a Watchface.

**Comment by zoraiz-WOL** [comment]
hi. how we can set the portrairt watchface time position to top and bottom

**Comment by Ankooo12** [comment]
@zoraiz-WOL  

if you want to change the position of time you have to change `timeElementUnitBaseline`

Example 
`  $0.imageList[0].timeElementUnitBaseline = 0.4 // bottom = 0.8 , top = 0.32 `

**Comment by Ankooo12** [comment]
Can you help me? in How to change the style of the time. I am changing in init of PortraitWatchface but it doesn't work in export.
